### PR TITLE
Troubleshoot build issues

### PR DESCRIPTION
**Pull request by jtpio**
Opening a draft PR to troubleshoot issues as seen in #1 

The "Check Release" workflow can be ignored for now as it might fail for other reasons: https://github.com/jupyter-server/jupyter_releaser/issues/170

### CODE REVIEW

**Comment by jtpio** [comment]
cc @DerThorsten maybe you would like to turn that check for now?

image

**Comment by DerThorsten** [comment]
jop, once sec =)

**Comment by DerThorsten** [comment]
I set it to `Require approval for first-time contributors who are new to GitHub` which should do the trick

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "build:dockerimage": "docker build -t mydockerimage  --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) . ",
     "build:emscripten": "echo $(pwd) && docker run --rm -v $(pwd):/src    -u $(id -u):$(id -g)  mydockerimage    ./build.sh",
     "copy-files": "copyfiles -u 1 src/xeus_lua.wasm  src/xeus_lua.worker.js src/xeus_lua.js lib",
-    "build:wasm": "jlpm run build:dockerimage && jlpm run build:emscripten && jlpm run copy-files",
+    "build:wasm": "jlpm run build:dockerimage && jlpm run build:emscripten",
```
Comment (jtpio) [inline package.json:35]: Thinking that we might not need to manually copy the files and `webpack` would take care of bundling them automatically. Let's see what CI says.

Comment (DerThorsten) [reply package.json:35]: When the copy-files is removed, I am also puzzled that xeus_lua.js does not end up in the "lib" folder.

Comment (jtpio) [reply package.json:35]: Gonna have a look locally.

**Comment by jtpio** [comment]
Also seeing the OOM errors locally with `node` 16:

```
$ jlpm run build:labextension:dev
yarn run v1.21.1
$ jupyter labextension build --development True .
Building extension in .

Compilation starting…


<--- Last few GCs --->

[872833:0x55e21a1d6ee0]   111428 ms: Scavenge 1962.0 (2078.9) -> 1953.9 (2078.9) MB, 17.7 / 0.0 ms  (average mu = 0.211, current mu = 0.145) allocation failure
[872833:0x55e21a1d6ee0]   111490 ms: Scavenge 1962.0 (2078.9) -> 1948.6 (2079.7) MB, 6.6 / 0.0 ms  (average mu = 0.211, current mu = 0.145) allocation failure
[872833:0x55e21a1d6ee0]   111556 ms: Scavenge 1962.7 (2079.9) -> 1949.4 (2080.2) MB, 5.5 / 0.0 ms  (average mu = 0.211, current mu = 0.145) task

```

**Comment by jtpio** [comment]
Locally excluding the emscripten generated files seems to help build the extension.

However the files are not properly emitted to `labextension/static` yet.

**Comment by jtpio** [comment]
@DerThorsten marked as ready in case you would like to get these changes in anyway

**Pull request merged**